src/realm/column_table.hpp

```diff
@@ -211,7 +211,11 @@ class SubtableColumn : public SubtableColumnBase {
 
     ConstTableRef get(size_t subtable_ndx) const
     {
-        return ConstTableRef(get_subtable_ptr(subtable_ndx));
+        int64_t ref = IntegerColumn::get(subtable_ndx);
```

Comment:
Can you add a small comment that because it's a *const* TableRef, it does not need a real persisted instance because it will not change. And that this is important when doing read-only iteration over lots of degenerate subtables in, for example, a query